### DISCUSSION TITLE
Dark Mode

### DISCUSSION THREAD

**Discussion by eduardochaves1** [xournalpp/xournalpp#3697] [Q&A] [answered]
Is there a way to use the app in a Dark Mode? I tried to look it into the settings but seem to don't have... (I'm a new user)

**Comment by rolandlo** [comment]
Yes, there is a way, but it must be compatible with your system's dark mode setting. Check the "Use dark theme" option in the view panel of menu `Edit > Preferences` and restart. 
What OS do you use and what version of Xournal++? Under Gnome based Linux distros you can switch to system dark mode in Gnome Tweaks (Appearance) for example.

**Reply by eduardochaves1** [reply]
I'm using the 1.1.0 version on my W10. I installed on my Linux Pop!_Os Gnome and worked fine with the dark theme. But on my W10, even doing the steps you said didn't work. After restarted my PC the checkbox was enabled but still white.

**Reply by LittleHuba** [reply]
I think you are understanding the "dark mode" incorrectly (We should rename this @rolandlo). 

We do not modify the appearance of the window itself. We solely modify the appearance of the icons and some small changes to the UI. Meaning, when "dark mode" is enabled, we use lighter icons so they are distinguishable from the dark background. How the window appears, depends completely on the system. You would have to look up how to switch the appearance of GTK apps to a "dark theme" on W10.

**Reply by eduardochaves1** [reply]
Ok, no problems. I can use in that way, hope someone can make a darker theme one day. Thanks anyway!

**Comment by Technius** [comment, marked as answer]
For Windows, you need to add a config file to force GTK to use the dark theme. See https://github.com/xournalpp/xournalpp/issues/1746#issuecomment-612533873 for more information. You'll also want to leave the confusingly named "Enable Dark Mode" option enabled so that dark mode icons are available.

This issue has been fixed in the nightly builds, so that "Enable Dark Mode" actually switches to the dark mode theme.

**Reply by eduardochaves1** [reply]
It worked! Thanks!
